lightning/src/onion_message/functional_tests.rs

```diff
@@ -418,6 +418,23 @@ fn one_blinded_hop() {
 	pass_along_path(&nodes);
 }
 
+#[test]
+fn blinded_path_with_dummy() {
+	let nodes = create_nodes(2);
+	let test_msg = TestCustomMessage::Pong;
+
+	let secp_ctx = Secp256k1::new();
+	let context = MessageContext::Custom(Vec::new());
+	let entropy = &*nodes[1].entropy_source;
+	let blinded_path =
+		BlindedMessagePath::new_with_dummy_hops(&[], 5, nodes[1].node_id, context, entropy, &secp_ctx).unwrap();
+	let destination = Destination::BlindedPath(blinded_path);
```

Comment:
Assert that the dummy hops are indeed added (hop count)?

Reply:
Done! [pr3728.02](https://github.com/shaavan/rust-lightning/commits/pr3728.02)
Thanks!